pkgs/by-name/cz/czkawka/package.nix

```diff
@@ -26,21 +37,16 @@ let
 
   self = buildRustPackage' {
     pname = "czkawka";
-    version = "7.0.0";
+    version = "8.0.0";
```

Comment:
Split the version and hash updates into their own commit.

Reply:
Done.